src/test/java/org/broadinstitute/hellbender/tools/GatherVcfsCloudIntegrationTest.java

```diff
@@ -92,7 +107,7 @@ private void assertGatherProducesCorrectVariants(GatherVcfsCloud.GatherType gath
     }
 
     @Test(dataProvider = "getHeaderOnlyCases")
-    public void testVcfsWithOnlyAHeader(List<File> inputs, final GatherVcfsCloud.GatherType gatherType, File expected){
+    public void testVcfsWithOnlyAHeader(List<File> inputs, final GatherVcfsCloud.GatherType gatherType, File expected) throws IOException {
         assertGatherProducesCorrectVariants(gatherType, expected, inputs);
     }
 
```

Comment:
Add a test for `IndexFeatureFile` proving that the tool can still index an unindexed `.vcf.gz` (if we don't already have a good test that does that)

Reply:
`IndexFeatureFileIntegrationTest.testVCFGZIndex_tabix()`